app/views/layouts/messages.html.erb

```diff
@@ -32,15 +31,13 @@
 
     <div id="right-column" class="flex flex-1 flex-col h-screen bg-white dark:bg-graphite relative">
 
-      <div id="handle-container" class="absolute left-0 top-1/2 w-8 h-20 -mt-10 z-20 hidden md:flex" data-click-toggle-target="trigger">
-        <div  id="handle"
-              class="flex-1 flex cursor-pointer group"
-              data-controller="click-toggle"
-              data-click-toggle-target="trigger"
-              data-click-toggle-flippable-class="!hidden"
-              data-action="keydown.meta+shift+s@document->click-toggle#handleEvent
-                          keydown.meta+period@document->click-toggle#handleEvent"
-              >
+      <div  id="handle-container"
+            class="absolute left-0 top-1/2 w-8 h-20 -mt-10 z-20 hidden md:flex"
+            data-action= "keydown.meta+shift+s@document->click-toggle#toggleClass
+                          keydown.meta+period@document->click-toggle#toggleClass
+                          click->click-toggle#toggleClass"
+      >
```

Comment:
I think we could put these keydown actions anywhere, but I figured we might as well group them with the click one.

Notably, because it's a div I had to explicitly specify `click->` but if it was a link or button then that's assumed.

Reply:
agreed, with the ability to dangle these things anywhere in the source it's worth putting them near their friends so they don't get lost.